src/main/index.ts

```diff
@@ -94,17 +94,21 @@ app.on("ready", async () => {
   windowManager = WindowManager.getInstance<WindowManager>(proxyPort);
 
   // call after windowManager to see splash earlier
-  const extensions = await extensionDiscovery.load();
+  try {
+    const extensions = await extensionDiscovery.load();
 
-  // Subscribe to extensions that are copied or deleted to/from the extensions folder
-  extensionDiscovery.events.on("add", (extension: InstalledExtension) => {
-    extensionLoader.addExtension(extension);
-  });
-  extensionDiscovery.events.on("remove", (lensExtensionId: LensExtensionId) => {
-    extensionLoader.removeExtension(lensExtensionId);
-  });
+    // Subscribe to extensions that are copied or deleted to/from the extensions folder
+    extensionDiscovery.events.on("add", (extension: InstalledExtension) => {
+      extensionLoader.addExtension(extension);
+    });
+    extensionDiscovery.events.on("remove", (lensExtensionId: LensExtensionId) => {
+      extensionLoader.removeExtension(lensExtensionId);
+    });
 
-  extensionLoader.initExtensions(extensions);
+    extensionLoader.initExtensions(extensions);
+  } catch (error) {
+    dialog.showErrorBox("Lens Error", `Could not load extensions: ${error?.message || ""}`);
```

Comment:
I don't think we should leave this blank if we don't have a message. The dangling `:` would make it look to me like something wasn't being handled correctly.

Reply:
Well, we know (at this time) that the error we're receiving is a proper error with message. Other usage in this file will actually throw if message is missing.. But yeah I guess I'll change this if we want to be that defensive.